vpr/src/place/compressed_grid.h

```diff
@@ -6,38 +6,98 @@
 #include "vtr_geometry.h"
 #include "vtr_flat_map.h"
 
-struct t_type_loc {
-    int x = OPEN;
-    int y = OPEN;
-
-    t_type_loc(int x_val, int y_val)
-        : x(x_val)
-        , y(y_val) {}
-
-    //Returns true if this type location has valid x/y values
-    operator bool() const {
-        return !(x == OPEN || y == OPEN);
-    }
-};
-
 struct t_compressed_block_grid {
+    //TODO: compressed_to_grid_x/y should become a 2d vector
```

Comment:
Add a top level comment on what the compressed grid is.  Explain x, y, layer assumptions / mapping.